lib/compiler.js

```diff
@@ -8,8 +8,6 @@ const shebangRegExp = /^#!.*/;
 const importExportRegExp = /\b(?:im|ex)port\b/;
 const getOption = require("./options.js").get;
 
-exports.parse = require("./parsers/default.js").parse;
-
```

Comment:
It looks like `compiler.parse` isn't being used. Since it's configurable in `compile` I opted for the easy route of simply removing the export.